app/lib/frontend/templates/views/account/consent.mustache

```diff
@@ -8,12 +8,6 @@
 {{& description_html}}
 </div>
 <p id="-admin-consent-buttons">
-  <button
-    id="-admin-consent-reject-button"
-    class="mdc-button mdc-button--raised pub-button-cancel"
-    data-mdc-auto-init="MDCRipple">Reject</button>
-  <button
-    id="-admin-consent-accept-button"
-    class="mdc-button mdc-button--raised"
-    data-mdc-auto-init="MDCRipple">Accept</button>
+  {{& reject_button_html}}
```

Comment:
If in the input we had written:
```
'reject_button': {
  'label': 'Reject'
  'id': '-admin-consent-reject-button',
  'classes': ['pub-button-cancel'],
},
'accept_button': { 
 ...
```
(instead of `'reject_button_html': renderButton(...`)

Then we could do use partial templates as follows:
```
{{#reject_button}}
  {{> button-template }}
{{/reject_button}}
{{#accept_button}}
  {{> button-template }}
{{/accept_button}}
```
Now in the `button-template.mustache ` file we have access to `{{label}}`, `{{id}}`, and `{{classes}}`.
I think that's quite simple and every powerful, if we don't want the reject button rendered we can just insert `null` and it doesn't get rendered at all..

----
We could also do something like:
```
{{#lambdas.render_button}}reject_button{{/lambdas.render_button}}
```
And then have the global render function inject a set of `lambdas` into all contexts..The lambda can just be a function that reads source and looks up `reject_button` reject button in source.

-----

I think partials is the better approach, especially if we can five the partials a name that indicates that they are files.. such that they are easy to find. The only danger with partials is that we don't get good errors if a variable is missing, but this is probably something we could fix in mustache (something we want to fix long term anyways).

Reply:
Correction, we use `lenient` mode right now, if we fixed that there would be a lot fewer concerns about missing the wrong variables..